Bugfix: Small fixes for various bugs

veronikaslc opened the pull request.

Commit:
[bugfix] Fixed CSS property naming to cammel-case

Commit:
[bugfix] Separated pedigree CSS property to exclude influence on other components

marta- reviewed (approved):
Looks good, didn't test

Commit:
[bugfix] Fix various bugs in Questionnaire
Fixed edit button icon
Fixed delete button
Some refactoring for unified code style

sashaandjic reviewed (approved):
tested
